src/Phinx/Config/Config.php

```diff
@@ -294,6 +304,52 @@ public function getTemplateClass()
      }
 
     /**
+     * Get the version order.
+     *
+     * @return string
+     */
+    public function getVersionOrder()
+    {
+        if (!isset($this->values['version_order'])) {
+            // if the configuration value is missing, return the default version order (creation time)
+            return self::VERSION_ORDER_CREATION_TIME;
```

Comment:
It would be nicer for the setter to have the guard (validate appropriate parameter, use default if not supplied and set/save it accordingly) and the getter be dumb, considering this is only exposing a single property in the config.

The setter would need to be called as part of the config instantiation process if it not already (commenting inline without seeing all the other code).